native/src/main/java/io/ballerina/persist/nodegenerator/CreateSyntaxTree.java

```diff
@@ -0,0 +1,204 @@
+/*
```

Comment:
Better if the file name doesn't begin with a verb. Shall we rename it to something like `SyntaxTreeGenerator`

Reply:
Updated in [8078951](https://github.com/ballerina-platform/persist-tools/pull/9/commits/8078951dff409765a286bc5e8f289d15b9b412ee)